src/tool/hpcrun/os/catamount/override.c

```diff
@@ -47,6 +47,7 @@
 #include <assert.h>
```

Comment:
we can delete this file and the directory containing it. catamount was a predecessor to cray's compute node linux.